src/components/Proposal/ProposalHistoryRow.tsx

```diff
@@ -207,27 +207,38 @@ const ConnectedProposalHistoryRow = connect<IStateProps, IDispatchProps, IContai
 
 export default (props: { proposal: Proposal; daoState: IDAOState; currentAccountAddress: Address}) => {
   const proposal = props.proposal;
-  const observable = combineLatest(
-    proposal.state(),
-    props.currentAccountAddress ? proposal.stakes({ where: { staker: props.currentAccountAddress}}) : of([]),
-    props.currentAccountAddress ? proposal.votes({ where: { voter: props.currentAccountAddress }}) : of([])
-  );
+  let observable;
+  if (props.currentAccountAddress) {
```

Comment:
this should be if !props.currentAccountAddress

Reply:
oops